trackers/core/reid/dataset/base.py

```diff
@@ -28,7 +34,7 @@ class TripletsDataset(Dataset):
     Attributes:
         tracker_id_to_images (dict[str, list[str]]): Dictionary mapping tracker IDs
             to lists of image paths
-        transforms (Optional[Compose]): Optional image transformations to apply
+        transforms (Union[Compose, ToTensor]): Image transformations to apply
```

Comment:
@onuralpszr what was the reason for making this change? The docstring no longer matches the actual type of the argument it describes.

Reply:
It does match and Previous code already enforcing  "Compose, ToTensor"  in transforms  I just added type annotation 

https://github.com/roboflow/trackers/blob/b28d819ffe7000af79e1ead39478f09707940940/trackers/core/reid/dataset/base.py#L47

And for the function side you return "Tensor" and I made sure it is Tensor (which is I transformed) but it is actually "Optional" I need to re-check